contrib/gabc.vim

```diff
@@ -1,7 +1,7 @@
 " Vim syntax file
 " Language:	Gabc gregorian chant notation
 " Maintainer:	Elie Roux <elie.roux@telecom-bretagne.eu>
-" Last Change:	2008 Nov 29
+" Last Change:	2016 Feb 11
```

Comment:
Looks good, I trust you've tested it better than I would, the only remark is the "Maintainer" Line that should be changed or removed